Newsletters: add 265 (2023-08-23)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Client/services section @bitschmidty 
- [x] Bitcoin Core 27460 @murchandamus

LarryRuane commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -0,0 +1,195 @@
+  proof.  The normal operation of the channel would include just one
+  extra step: after each update to the channel, Alice gives Bob a
+  signature over the current state (which includes a state number).
+  Then, each time Alice organically reconnects to Bob, she requests the
```
```suggestion
  Then, each time Alice organically reconnects with Bob, she requests the
```

LarryRuane commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -0,0 +1,195 @@
+    version experimentally implemented in Core Lightning earlier this
+    year (see [Newsletter #238][news238 peer storage]), and (according
+    to a message from Bastien Teinturier [indicates][teinturier
+    backups]) a verson of the scheme implemented in the Phoenix wallet
```
```suggestion
    backups]) a version of the scheme implemented in the Phoenix wallet
```

LarryRuane commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -0,0 +1,195 @@
+  that data before accepting new data from Bob.  If Bob also had a lot
+  of data queued to send to Alice's node, he also wouldn't accept new data
+  from Alice.  This could lead to neither of them attempting to receive
+  data from the other indefinitely.  The issue was original discovered
```
```suggestion
  data from the other indefinitely.  The issue was originally discovered
```

murchandamus commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -174,7 +174,9 @@ Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
 [Bitcoin Inquisition][bitcoin inquisition repo].*
 
-- [Bitcoin Core #27460][] rpc: Add importmempool RPC FIXME:Murchandamus
+- [Bitcoin Core #27460][] adds a new `importmempool` RPC. The RPC will
+  load a `mempool.dat` file and attempt to add the loaded transactions
+  to its mempool.
```
I went back and forth with myself, whether it is useful to mention the options to the broader audience, but I didn’t have a good idea how to make it relevant. My closest attempt was to say:

“The RPC has options to import some transaction metadata from the imported file that may be useful to mining pool operators or other power users.”

But I was only half convinced. Any thoughts?

glozow replied:
fwiw I don't really see those options being useful to someone who isn't a power user + trying to replicate something very specific on a test node

harding replied:
If y'all are unconvinced, I'm unconvinced.  Dropping it from this week's newsletter.  If anyone has second thoughts, we can always cover it next week.

glozow replied:
Ah just to clarify, I didn't mean that the summary should be dropped, just that a description of the options didn't need to be added. I think the RPC in general is worth covering!

MarnixCroes commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -0,0 +1,245 @@
+  Sparrow [1.7.8][sparrow 1.7.8] added support for [BIP322][] [message signing][topic generic
+  signmessage], including P2TR addresses, and made various improvements to
+  [RBF][topic rbf] and [CPFP][topic cpfp] fee bumping features.
+
```
```suggestion

- **Wasabi Wallet 2.0.4 released:**
  Wasabi [2.0.4][wasabi 2.0.4] includes a daemon, TurboSync (for faster wallet load), 
  privacy suggestions when sending, a _Speed Up Transaction_ button for increasing the
  fee of an unconfirmed transaction with [RBF][topic rbf] or [CPFP][topic cpfp], a new 
  amount decomposer for the coinjoins and lots of other improvements and bug fixes.
```

bitschmidty replied:
@MarnixCroes thanks for the heads up. Reworded a bit and added here: https://github.com/bitcoinops/bitcoinops.github.io/pull/1250/commits/c3d8c4a97f756b0bd506dbda49c66103bc9c6a39

MarnixCroes commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -0,0 +1,245 @@
+[torq github]: https://github.com/lncapital/torq
+[torq blog]: https://ln.capital/articles/announcing-torq-V1.0
+[blixt v0.6.8]: https://github.com/hsjoberg/blixt-wallet/releases
+[sparrow 1.7.8]: https://github.com/sparrowwallet/sparrow/releases/tag/1.7.8
```
```suggestion
[sparrow 1.7.8]: https://github.com/sparrowwallet/sparrow/releases/tag/1.7.8
[wasabi 2.0.4]: https://github.com/zkSNACKs/WalletWasabi/releases/tag/v2.0.4
```

MarnixCroes reviewed (commented):
new Wasabi release yesterday https://github.com/zkSNACKs/WalletWasabi/releases/tag/v2.0.4

bitschmidty commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -0,0 +1,251 @@
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27460][] adds a new `importmempool` RPC. The RPC will
```
This might be helpful for readers as well
https://bitcoincore.reviews/27460

bitschmidty commented:
Note: Travis CI is not running and Im working with support to get this resolved.

harding commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -0,0 +1,195 @@
+
+## News
+
+- **Fraud proofs for outdated backup state:** Thomas Voegtlin
```
Leaving as-is.

harding commented:
Addressed review comments (thanks @LarryRuane @murchandamus @glozow !), reviewed contributions from @bitschmidty and @murchandamus (thanks!), added lede, releases/RCs, and topic entries.  Also rand the production tests locally (they passed).

THanks everyone!

azuchi commented on _posts/en/newsletters/2023-08-23-newsletter.md:
```diff
@@ -0,0 +1,249 @@
+lang: en
+---
+This week's newsletter describes fraud proofs for outdated backup state
+and includes our regular sections summaring recent changes to services
```
```suggestion
and includes our regular sections summarizing recent changes to services
```

bitschmidty replied:
Thanks for catching this @azuchi !

bitschmidty commented:
CI fixed. 
Pushed a fixup for @azuchi feedback (https://github.com/bitcoinops/bitcoinops.github.io/pull/1250#discussion_r1302323580) and a topic wording fixup.
Squashed and merged.

Thank you @harding @LarryRuane @murchandamus @MarnixCroes @azuchi @glozow for contributing this week! 🚀
